Base/usr/share/man/man8/useradd.md

```diff
@@ -0,0 +1,44 @@
+## Name
+
+useradd - add a new user to the system password file
+
+## Synopsis
+
+```**sh
+# useradd [options] LOGIN
+```
+
+## Description
+
+This program uses the interface defined `pwd.h` to add a new user to the system
+password file. 
+
+A user can either be a `login` user (that is, they have a home
+directory; someone who actually uses to system) or a `system` user (such
+as the `lookup` user found in `/etc/passwd`). 
```

Comment:
Is any of this relevant to the `useradd` utility? I don't have to specify whether the account I'm creating is a `login` or `system` user, so why are we mentioning it at all? This text seems more suited for a man page describing how user accounts work in general.

Reply:
I suppose not. I'll remove it :)